README.md

```diff
@@ -171,7 +170,7 @@ Automatically sourced if you use a Smartcar-hosted redirect.
 #### Example
```

Comment:
I can't comment on this, but the description of `onComplete` in the section above is incorrect

Reply:
ah okay, makes sense. The part that I find misleading is it it states *your* redirect URI. Where technically for the hosted on, it really isn't *their* redirect URI.  I'll leave this as a nit